lib/azure/armrest/model/storage_account.rb

```diff
@@ -35,6 +36,37 @@ def containers(key = nil)
         end
       end
 
+      # Returns the properties for the given container +name+ using account +key+.
+      # If no key is provided, it is assumed that the StorageAccount object
+      # includes the key1 property.
+      #
+      def container_properties(name, key = nil)
+        key ||= properties.key1
+
+        response = blob_response(key, "restype=container", name)
+
+        ContainerProperty.new(response.headers)
+      end
+
+      # Returns the properties for the given container +name+ using account +key+.
+      # If no key is provided, it is assumed that the StorageAccount object
+      # includes the key1 property.
+      #
+      # If the returned object does not contain x_ms_blob_public_access then
+      # the container is private to the account owner. You can also use the
+      # :private? method to determine if the account is public or private.
+      #
+      def container_acl(name, key = nil)
+        key ||= properties.key1
+
+        response = blob_response(key, "restype=container&comp=acl", name)
+
+        obj = ContainerProperty.new(response.headers)
```

Comment:
You mean like this?

    response.headers[:private?] = response.headers.include?(:x_ms_blob_public_access) ? false : true

I can do that, though I'm curious why it matters.